docs/static/monitoring/monitoring-apis.asciidoc

```diff
@@ -487,6 +491,28 @@ Example response:
         "out" : 216485,
         "queue_push_duration_in_millis" : 342466
       },
+      "flow" : {
+        "input_throughput" : {
+          "current": 189.720,
+          "lifetime": 201.841
+        },
+        "filter_throughput" : {
+          "current": 187.810,
+          "lifetime": 201.799
+        },
+        "output_throughput" : {
+          "current": 191.087,
+          "lifetime": 201.761
+        },
+        "queue_backpressure" : {
+          "current": 0.277,
```

Comment:
can we also mention the range particularly for backpressure? I feel users can set up a threshold to get the alarms. Or if we will have a chance to integrate in APM, worth mentioning alarm feature (ES watchers may be helpful?!)

Reply:
We most certainly need to document it.

For pipelines that are being _pushed_ data, effectively the only good number is zero.

The magnitude of value is highly dependent on the shape of the pipeline's inputs, and is _not_ comparable from one pipeline to another. A pipeline with a single-threaded input (like stdin, pipeline) will have a range of 0-1 (what percent of wall-clock time is that one thread blocked), while a pipeline with inputs that have many inbound connections will have a much higher range (what percentage of wall-clock time are _each_ of those connections blocked, _added together_ -- 100 inbound connections that are each blocked 75% of the time will have a value of about `75.0`).

Additionally, with inputs that are _pulling_ data (local file input, jdbc input, elasticsearch input, http poller, etc), some amount of queue back-pressure is normal and expected, and results in the input pulling data a little slower but _without losing its place_.